resources/menus/number_of_deuces.xml

```diff
@@ -0,0 +1,4 @@
+<menu id="DeucesMenu" title="Deuces">
+    <menu-item id="deuces_1" label="At 1st - Default" />
```

Comment:
i'm thinking that maybe instead of having menu for

* golden point vs advantage
* 1 deuce vs 2 deuce

we should simply have a menu with 3 options

* **golden point** vs **advantage** vs **amateur golden point**

this would change the logic a bit.  but would avoid a user having to select among many menus

will comment in place

Reply:
IMHO, this make sense if you plan to fix #27 
So in this way you have only one page where you can select the "game mode" that is actually a preset and in the core file you'll have only sections, one for each "game mode"
Sounds good, but the core has to be rewritten from scratch I think
Anyway, why 1 deuce vs 2 deuce? In "amateur" mode is a kind of Golden Point extended, that is not present in advantage mode, that is more a Tennis mode. Do you normally use advantage with some limit?

Reply:
i think we'll need to fix #27 , otherwise it will be hard for users to select among so many options

i barely use advantage mode. official portuguese tournament use golden point, so all unofficial tournaments follow barely the same rules, and friendly matches as well. there are some tournaments all-vs-all where each match is time-based, and others that do use "pro-set", which if I recall correctly is a single set until 9 matches

Reply:
Like I said, yes, make more sense to me.
How we can proceed? One file for each game-mode? With one file for every game mode all became easy to update, easy to work with and easy to debug.
Or do you prefer one big master file with all inside?